compass/conf/CoordinatorState.java

```diff
@@ -0,0 +1,42 @@
+/*
+ * This file is part of TestnetCOO.
+ *
+ * Copyright (C) 2018 IOTA Stiftung
+ * TestnetCOO is Copyright (C) 2017-2018 IOTA Stiftung
+ *
+ * TestnetCOO is free software: you can redistribute it and/or modify
+ * it under the terms of the GNU Affero General Public License as published
+ * by the Free Software Foundation, either version 3 of the License,
+ * or (at your option) any later version.
+ *
+ * TestnetCOO is distributed in the hope that it will be useful,
+ * but WITHOUT ANY WARRANTY; without even the implied warranty of
+ * MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE.  See the
+ * GNU Affero General Public License for more details.
+ *
+ * You should have received a copy of the GNU Affero General Public License
+ * along with TestnetCOO.  If not, see:
+ *      http://www.gnu.org/licenses/
+ *
+ * For more information contact:
+ *     IOTA Stiftung <contact@iota.org>
+ *     https://www.iota.org/
+ */
+
+package org.iota.compass.conf;
+
+import jota.model.Transaction;
+
+import java.io.Serializable;
+import java.util.List;
+
+public class CoordinatorState implements Serializable {
+
+  public static final String COORDINATOR_STATE_PATH = "compass.state";
+
+  public Integer latestMilestoneIndex;
+  public String latestMilestoneHash;
+  public long latestMilestoneTime;
+  public List<Transaction> latestMilestoneTransactions;
+
```

Comment:
I suppose if compass will stay a small codebase it is fine (and even less verbose) to make everything publicly scoped like that

Reply:
It is just like the config object really.